patch/0015-mlxsw-core-thermal-Enable-thermal-zones-by-default.patch

```diff
@@ -0,0 +1,48 @@
+From 21833c9ab0b71b921f830e9a338b2084a0787a7e Mon Sep 17 00:00:00 2001
+From: Vadim Pasternak <vadimp@mellanox.com>
+Date: Thu, 21 May 2020 13:41:23 +0300
+Subject: [backport 4.19 3/3] mlxsw: core: thermal: Enable thermal zones by
+ default
+
+Change default setting for the all thermal zones from "disabled" to
```

Comment:
what is the upstream commit id?

Reply:
Hi Guohan, actually these codes are already in upstream, previously when backporting to SONiC this was set to disable because of the thermal control not available,  here we just restore it back, so no new code added here.

Reply:
@lguohan is it ok to be merged?
by accident i merged the dependent one before this one was taken properly to buildimage.

Reply:
what is the upstream commit id? can you put it into the patch message?